episodes/15-explanation.md

```diff
@@ -33,62 +20,94 @@ exercises: 40
-  But, as a general rule,
-  it is better to assume that you are under-estimating
-  rather than over-estimating the length.
-  Experience suggests that that is most likely correct,
-  and anyway it is better to have spare time for discussion,
-  revisiting exercises, etc
-  than to try to rush through the remaining material in limited time.
-  - If, after teaching your new lesson, you find that you did not have time to cover all the content,
-    approach cutting down the lesson by identifying which learning objectives to remove.
-    Then take out the objective(s) and the corresponding assessment(s) and explanatory content.
-- Consider adding suggestions for objectives that can be skipped to the Instructor Notes for the lesson.
-- Managing cognitive load
+## Less is More
+
+Trying to fit too much content into a lesson is counter-productive.
+It is better to cover less and provide a smaller but stable foundation for learners to build upon.
+
+You should also consider a realistic workshop length for teaching your lesson.
+It is difficult to keep learners attentive for many hours over many days.
+Most Carpentries workshops are two work days of lesson material as it is difficult for learners to
+attend 4 full days of a workshop.
+Although, there are some in the community who will run workshops that are three days to cover additional material.
+
+As you consider the length of your lesson discuss with your collaborators and ask yourself: 
+
+- What is essential to include?
+- What can be left out if needed?
+- Are there checkpoints where the lesson could end if needed?
+- Can important concepts be moved up earlier to ensure they are covered?
+
+
+In the end, the only way to know for sure is to teach the lesson, measuring how long it takes to teach.
+Borrowing from television, the Carpentries community calls these early workshops, pilot workshops.
+As you run pilot workshops, you can note the length of time spend in each episode.
+You might find the [template for notes on pilot workshops][pilot-notes-template] helpful as it includes a table for episode and exercise timings.
+Instructors commonly report running short on time in workshops.
+Thus, it is better to assume that you are under-estimating rather than over-estimating the length.
+It is better to have additional time for discussion, review, and wrap-up than to rush through material to fit it into the remaining time.
+
+If, after piloting your new lesson, you find that you did not have time to cover all the 
+content, approach cutting down the lesson by identifying which learning objectives to 
+remove. Then take out the objective(s) and the corresponding assessment(s) and explanatory
+content. You may consider making a [concept map][mental-map-instructor-training] to help identify which objectives depend on one another.
+
+Alternatively, if you decide to keep certain objectives in the lesson, you can add suggestions on which objectives can be skipped to the Instructor Notes for the lesson.
```

Comment:
Have we introduced Instructor Notes yet? If not, I guess this is the place to do it.